csss-site/src/events/workshops/templates/workshops/main.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+{% extends "csss/header.html" %}
+{% block content %}
+{{ block.super }}
+        <h2 class="title is-size-2" style="padding-left: 1em;">Computing Science Student Society Workshops</h2>
+        <div style="padding-left: 2em;">Note: Most of these workshop slides were part of various levels of demos. Parts of slides may be left empty or leave questions unanswered for that reason.</div><br><br>   
+
+        <h3 class="title" style="padding-left: 1em;">Git Workshops</h3>
+	<div style="padding-left: 2em;">Learn about Git and GitHub, what they are, how to use them in both the terminal and the Visual Studio Code.</div><br>
+        <div style="text-align:center;">
+		<iframe src="https://drive.google.com/file/d/1jWSagoGBFAAf87LOFIOediP_L3bFFf6K/preview" width="800" height="600" allow="autoplay"></iframe>
```

Comment:
this link is broken, go to https://drive.google.com/file/d/1jWSagoGBFAAf87LOFIOediP_L3bFFf6K/view and see

Reply:
Fixed in 6ff9b6e71afda55b5fdd34e6251d596530f6f264